Content.Server/Traitor/Systems/AutoTraitorSystem.cs

```diff
@@ -0,0 +1,84 @@
+using Content.Server.GameTicking.Rules;
+using Content.Server.Mind;
+using Content.Server.Mind.Components;
+using Content.Server.Traitor.Components;
+using Robust.Shared.Map;
+
+namespace Content.Server.Traitor.Systems;
+
+/// <summary>
+/// Makes entities with <see cref="AutoTraitorComponent"/> a traitor either immediately if they have a mind or when a mind is added.
+/// </summary>
+public sealed class AutoTraitorSystem : EntitySystem
+{
+    [Dependency] private readonly IMapManager _map = default!;
+    [Dependency] private readonly MindSystem _mind = default!;
+    [Dependency] private readonly TraitorRuleSystem _traitorRule = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AutoTraitorComponent, ComponentStartup>(OnStartup);
+        SubscribeLocalEvent<AutoTraitorComponent, MapInitEvent>(OnMapInit);
+        SubscribeLocalEvent<AutoTraitorComponent, MindAddedMessage>(OnMindAdded);
+    }
+
+    // workaround for #19409
+    private void OnStartup(EntityUid uid, AutoTraitorComponent comp, ComponentStartup args)
```

Comment:
I will just code this you noob.

Reply:
ok troller say when it is :trollface:

Reply:
@metalgearsloth :trollface:

Reply:
I got like 10 layers deep of soap but still working on it.

Reply:
pro